package.json

```diff
@@ -29,6 +29,7 @@
     "react-router": "^6.0.0-beta.0",
```

Comment:
adding comment here, but do you know why ther eis 36,000 additions to `package-lock.json` 🤔

Reply:
hmm I honestly have no clue 😕 tried searching a bit but I really don't understand it enough to know how it could have 36k+ additions?